internal/dialects/spanner.go

```diff
@@ -0,0 +1,50 @@
+package dialects
+
+import (
+	"fmt"
+
+	"github.com/pressly/goose/v3/database/dialect"
+)
+
+// NewSpanner returns a [dialect.Querier] for Spanner dialect.
+func NewSpanner() dialect.Querier {
+	return &spanner{}
+}
+
+type spanner struct{}
+
+var _ dialect.Querier = (*spanner)(nil)
+
+func (s *spanner) CreateTable(tableName string) string {
```

Comment:
This is the only implementation that doesn't have an auto-incrementing `id` column. But I suspect this is fine, it was only necessary for historic reasons anyways, and we never returned this to users.

You already created the `version_id` as the PK, so this is probably fine.